.github/workflows/build.yml

```diff
@@ -49,54 +49,40 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+      - name: Checkout Picasso repository
+        uses: actions/checkout@v4
+        with:
+          repository: edunext/picasso
+          ref: MJG/scripts-folder
```

Comment:
We should replace the line before merge to point main branch. 

The rest looks good to me, thanks :)

Reply:
Done, thank you!